korgan/claim_release_fail_closed.py

```
@@ -0,0 +1,98 @@
+    """
+    global _INSTALLED
+    if _INSTALLED:
+        return
+
+    from korgan import universal_claim_runtime as runtime
+    from korgan import bot as base_bot
+
+    original: Callable[..., Awaitable[None]] = runtime._send_claim
+
+    async def guarded_send_claim(
+        message: Any,
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `message`

(ANN401)